webapp/sources/api-doc/components/responses/compliance-directive-id.yml

```diff
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: CC-BY-SA-2.0
+# SPDX-FileCopyrightText: 2013-2023 Normation SAS
+type: object
+required:
+  - result
+  - action
+  - data
+properties:
+  result:
+    type: string
+    description: Result of the request
+    enum:
+      - success
+      - error
+    example: success
+  action:
+    type: string
+    description: The id of the action
+    enum:
+      - getDirectiveComplianceId
+  data:
+    type: object
+    required:
+      - directiveCompliance
+    properties:
+      directiveCompliance:
+        type: object
+        required:
+          - id
+          - name
+          - mode
+          - compliance
+          - complianceDetails
+          - rules
+          - nodes
+        properties:
+          id:
+            type: string
+            format: uuid
+            description: id of the directive
+            example: 9a1773c9-0889-40b6-be89-f6504443ac1b
+          name:
+            type: string
+            description: name of the directive
```

Comment:
you should decide to either start description with an uppercase or not and be consistent with it. IIRC this docs has uppercase.